.github/workflows/build.yml

```
@@ -0,0 +1,61 @@
+          server-password: '${{ secrets.GITHUB_TOKEN }}'
+          overwrite-settings: true
+          job-status: 'success'
+          token: '${{ secrets.GITHUB_TOKEN }}'
+          
+      - name: Cache SonarQube packages
+        uses: actions/cache@v4
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)